src/utils.py

```diff
@@ -1,3 +1,51 @@
 '''
 Contains any and all code we didnt want to put somewhere else
-'''
\ No newline at end of file
+'''
+import numpy as np
+import librosa
+import hyperparameters as hp
```

Comment:
What are our thoughts on hyperparameters?  I was considering using a txt file and arg parse, but I kind of like the simplicity of just having a python file with them...

Reply:
I added support for a json file for args. The python file is only good for consts, so I renamed it to audio_parameters, and moved actual training hyperparameters to train.json as an example. That way we can just make new json files when training under new parameters.

Reply:
Love it!